### PR TITLE
update sync-diff-inspector: update settings docs

### PR DESCRIPTION
**Pull request by Leavrth**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

`Table configs` of settings has been updated.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tidb-tools/pull/546 & pingcap/docs-cn#7353

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sync-diff-inspector/sync-diff-inspector-overview.md [hunk 1]
```diff
@@ -155,8 +137,25 @@ target-table = "t2" # 目标表名
...
+
+######################### Table config #########################
+# 对部分表进行特殊的配置，配置的表必须包含在 task.target-check-tables 中
+[table-configs.config1] # config1 是该配置的唯一标识 id，用于上面 task.target-configs 中
+# 目标表名称，可以使用正则来匹配多个表，但不允许存在一个表同时被多个特殊配置匹配。
+target-tables = ["schema*.test*", "test2.t2"]
+# 指定检查的数据的范围，需要符合 sql 中 where 条件的语法
+range = "age > 10 AND age < 20"
+# 指定用于划分 chunk 的列，如果不配置该项，sync-diff-inspector 会选取一些合适的列（主键／唯一键／索引）
+index-fields = ["col1","col2"]
```
Comment (Leavrth) [inline sync-diff-inspector/sync-diff-inspector-overview.md:151]: 更新1，index-fields格式

File: sync-diff-inspector/sync-diff-inspector-overview.md [hunk 2]
```diff
@@ -155,8 +137,25 @@ target-table = "t2" # 目标表名
...
 
-    # 对部分表的额外配置
+    # 对部分表的额外配置，其中 config1 在下面 Table config 配置栏中定义
     target-configs= ["config1"]
+
+######################### Table config #########################
+# 对部分表进行特殊的配置，配置的表必须包含在 task.target-check-tables 中
+[table-configs.config1] # config1 是该配置的唯一标识 id，用于上面 task.target-configs 中
+# 目标表名称，可以使用正则来匹配多个表，但不允许存在一个表同时被多个特殊配置匹配。
+target-tables = ["schema*.test*", "test2.t2"]
```
Comment (Leavrth) [inline sync-diff-inspector/sync-diff-inspector-overview.md:147]: 更新2，target-tables 格式

Comment (Leavrth) [reply sync-diff-inspector/sync-diff-inspector-overview.md:147]: 以及注释

File: sync-diff-inspector/sync-diff-inspector-overview.md (hunk 1 above)
Comment (Leavrth) [reply sync-diff-inspector/sync-diff-inspector-overview.md:151]: 以及注释

**Review by Liuxiaozhen12 — APPROVED** [review]
LGTM

**Review by 3pointer — APPROVED** [review]
LGTM

**Pull request merged**